bin/install-wp-tests.sh

```diff
@@ -58,12 +58,22 @@ set -ex
 install_wp() {
 
 	if [ -d $WP_CORE_DIR ]; then
-		return;
+		while true; do
+			read -p "The $WP_CORE_DIR directory already exists. Overwrite it? (y/n) " yn
+			case $yn in
+				y|Y ) rm -rf $WP_CORE_DIR; break;;
+				n|N ) exit;;
+				* ) echo invalid response;
```

Comment:
Should we make `invalid response` a string?

Reply:
I added quotes and updated the message to be a bit more descriptive.